src/app/researcher/study/request/[memberIdentifier]/study-document-files.tsx

```diff
@@ -0,0 +1,42 @@
+'use client'
+
+import React, { FC } from 'react'
+import { Badge, Stack, Text } from '@mantine/core'
+import { useQuery } from '@tanstack/react-query'
+import { Download } from '@phosphor-icons/react/dist/ssr'
+import { dataForStudyDocumentsAction } from '@/server/actions/study-job.actions'
+export const StudyDocuments: FC<{ studyId: string }> = ({ studyId }) => {
+    const { data, isLoading } = useQuery({
+        queryKey: ['studyDocuments', studyId],
+        queryFn: () => dataForStudyDocumentsAction(studyId),
+    })
+
+    if (isLoading) return null
+
+    if (!data || data.documents.length === 0) {
+        return <Text>No documents found!</Text>
+    }
+
+    interface Document {
+        name: string
+        path: string
+    }
+
+    const documentChips = data.documents.map((doc: Document) => (
+        <Badge
+            key={doc.path}
+            color="#D4D1F3"
+            c="black"
+            component="a"
+            href={`/dl/study/${data.studyInfo.memberIdentifier}/${studyId}/docs/${doc.path}`}
```

Comment:
we need to always use the lib/path.ts helpers for determining the path to a file